tests/lambda_endpoints/test_ialirt_ingest.py

```diff
@@ -105,3 +114,48 @@ def test_parse_packet_s3(mock_packet_file_to_datasets, s3_test_packet, tmp_path)
     # Check if file was downloaded
     real_tmp_file = tmp_path / filename
     assert real_tmp_file.exists()
+
+
+def test_query_filenames(s3_client):
+    """Test the query_filenames function."""
+    bucket = "test-data-bucket"
+    region = "us-west-2"
+
+    fixed_now = datetime(2025, 4, 28, 16, 5, 0, tzinfo=timezone.utc)
+
+    test_key = "packets/iois_1_packets_2025_118_16_04_00"
+    s3_client.put_object(Bucket=bucket, Key=test_key, Body=b"test file")
+
+    filenames = query_filenames(bucket, region, fixed_now)
+
+    assert test_key in filenames
```

Comment:
I think you can be more explicit here and that you only expect this one item to be present.
```suggestion
    assert filenames == [test_key]
```